Sources/Dependencies/WithDependencies.swift

```diff
@@ -338,15 +338,15 @@ public func withEscapedDependencies<R>(
 public func withEscapedDependencies<R>(
   _ operation: (DependencyValues.Continuation) async throws -> R
 ) async rethrows -> R {
-  try await operation(DependencyValues.Continuation())
+  try await operation(DependencyValues.Continuation(dependencies: ._current))
 }
 
 extension DependencyValues {
   /// A capture of dependencies to use in an escaping context.
   ///
   /// See the docs of ``withEscapedDependencies(_:)-5xvi3`` for more information.
   public struct Continuation: Sendable {
-    @Dependency(\.self) private var dependencies
+    let dependencies: DependencyValues
```

Comment:
Could also localize the capture here:

```suggestion
    let dependencies = DependencyValues._current
```

Reply:
I'll rework the latest commit in this direction.